src/screens/AnthropometricScreen.js

```diff
@@ -0,0 +1,170 @@
+
+
+const AnthropometricScreen = () => {
+    const [weight,setWeight] = useState('');
+    const [height,setHeight] = useState('');
+    const [bmi,setBmi] =  useState('');
+    const [bmiAssessment,setBmiAssessment] =  useState('');
+    const [activityLevel,setActivityLevel] = useState('');
+    const [activityLevelValue,setActivityLevelValue] = useState(0);
+    const [DBW,setDBW] = useState(0);
+    const [TEA,setTEA] = useState(0);
+    const [distributions,setDistributions] = useState({
+        carbsCalorie: 0, 
+        proteinsCalorie: 0,
+        fatsCalorie: 0,
+        carbs: 0,
+        proteins: 0,
+        fats: 0,
+        riceExchange: 0,
+        meatAndFishExchange: 0,
+        fatExchange: 0
+    });
+    const activityDescription = "Activity Level\n Bedrest (but mobile; hospital patients)\n Sedentary (mostly sitting)\n Light (tailor, nurse, physician, jeepney driver)\n Moderate (carpenter, painter, heavy housework)\n Very Active (swimming, lumberman, athlete)";
+
+    const computeBMI = () => {
+        const pheight = Math.pow(parseFloat(height)/100,2);
+        const result = Math.ceil((parseFloat(weight)/pheight));
+        setBmi(result.toString());
+        computeActivityLevel();
+        if(result<18.5){
+            setBmiAssessment('Underweight');
+        }
+        else if(result >= 18.5 && result <= 24.9){
+            setBmiAssessment('Normal');
+        }
+        else if(result >= 25 && result <= 29.9){
+            setBmiAssessment('Overweight');
+        }
+        else if(result > 30){
+            setBmiAssessment('Obese');
+        }
+        
+    }; 
+
+    const computeActivityLevel = () => {
+        var value = 0;
+
+        switch(activityLevel){
+            
+            case 'Bedrest':
+                value = 27.5;
+                break;
+            case 'Sedentary':
+                value = 30;
+                break;
+            case 'Light':
+                value = 35;
+                break;
+            case 'Moderate':
+                value = 40;
+                break;
+            case 'Very Active':
+                value = 45;
+                break;
+            default:
+                value = 35;
+                break;
+        }
+        setActivityLevelValue(value);
+        setTEA((weight*value));
+    };
+
+    useEffect( () => {
+        setDistributions({
+            carbsCalorie: TEA*0.65,
+            proteinsCalorie: TEA*0.15,
+            fatsCalorie: TEA*0.2,
+            carbs: (TEA*0.65)/4,
+            proteins: (TEA*0.15)/4,
+            fats: (TEA*0.2)/9,
+            riceExchange: Math.round(((Math.ceil(((TEA*0.65)/4)/5)*5)-83)/23),
+            meatAndFishExchange: Math.round(((Math.ceil(((TEA*0.15)/4)/5)*5)-24)/8),
+            fatExchange: Math.round(((Math.ceil(((TEA*0.2)/9)/5)*5)-19)/5),
+        });
+    },[TEA]);
+
+
+
+    return ( <ScrollView>
+        <Text style={styles.text}>Weight in kg</Text>
```

Comment:
Add white space to separate sections: (i.e. <Text><Input> [white space] <Text><Input>)